gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/GitFlowGraphMonitor.java

```diff
@@ -128,10 +131,11 @@ void processGitConfigChanges() throws GitAPIException, IOException {
 
   static class GitFlowgraphComparator implements Comparator<DiffEntry>, Serializable {
     public int compare(DiffEntry o1, DiffEntry o2) {
-      Integer o1Depth = (o1.getNewPath() != null) ? (new Path(o1.getNewPath())).depth() : (new Path(o1.getOldPath())).depth();
-      Integer o2Depth = (o2.getNewPath() != null) ? (new Path(o2.getNewPath())).depth() : (new Path(o2.getOldPath())).depth();
+      Integer o1Depth =
```

Comment:
Might be because I'm not too familiar with this, but could we change o1Depth and o2Depth to be a bit more descriptive? Not too sure what o1 and o2 stand for